src/lib/user/user.ts

```diff
@@ -10,17 +13,80 @@ export abstract class User implements DefaultUser {
 		)!;
 	}
 
-	public static changeUserClass(userClass: new (ref: Player | number) => User): void {
+	async load(): Promise<unknown> {
+		return ""; /** What it is "loading" itself? And from "where"? */
+	}
+
+	async unload(): Promise<void> {}
+}
+
+class OfflineUser extends User implements OfflineUserDeclaration {
+	release(): void {
+		/** Somehow, release the offline user if no longer needed */
+	}
+}
```

Comment:
Just lock it and prevent it from being used.

Reply:
What do you mean by this?